README.md

```diff
@@ -73,6 +74,12 @@ $ deprecation-detector check src/ composer.lock
 $ deprecation-detector check src/ .rules/some_generated_rule_set
 ```
 
+If you have different source paths, you can pass them seperated by a comma.
+
+```bash
+$ deprecation-detector check src/,legacy/ vendor/
```

Comment:
You don't need the trailing slash I usually add them to make clear that it is a directory